OpenSim/Simulation/StatesTrajectory.h

```diff
@@ -414,6 +430,12 @@ class OSIMSIMULATION_API StatesTrajectory {
      * exactly reproduce results from the initial state trajectory; constraints
      * may not be satisfied, etc.
      *
+     * The states in the resulting trajectory will be realized to
+     * SimTK::Stage::Instance. You should not use the resulting trajectory with
+     * an instance of the model other than the you pass to this function; the
+     * state contains Instance-stage cache variables that point to force
```

Comment:
If it is just forces with raw pointers to `SimTK::Force`s we could disentangle so it works as expected. Perhaps post 4.0